charts/k8ssandra/templates/grafana/system-metrics.configmap.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.monitoring.grafana.provision_dashboards }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-k8ssandra-grafana-system-metrics-dashboard
```

Comment:
What do you think about shortening the name to `{{ .Release.Name }}-cassandra-overview`?

Reply:
> LGTM! I'd consider moving the configmap templates into the directories that correspond to the components they monitor, e.g. `templates/grafana/stargate.configmap.yaml` could move to `templates/stargate/grafana-dashboard.yaml`.

This is a good suggestion. @jsanda what are your thoughts on getting rid of the grafana folder altogether?